### PR TITLE
 Add SMAIL GPL Licence with rules and tests #928

### PR DESCRIPTION
**Pull request by techytushar**
PR for issue #928 
Signed-off-by: Tushar Mittal <chiragmittal.mittal@gmail.com>

### CODE REVIEW

File: src/licensedcode/data/licenses/smail_gpl.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+key: smail_gpl
+short_name: SMAIL GPL
+name: SMAIL GENERAL PUBLIC LICENSE
+category: Copyleft
```
Comment (techytushar) [inline src/licensedcode/data/licenses/smail_gpl.yml]: @pombredanne @DennisClark please check the license category.

File: src/licensedcode/data/licenses/smail_gpl.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+key: smail_gpl
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/smail_gpl.yml]: Category is good. Id-wise I would prefer `smail-gpl`

File: src/licensedcode/data/licenses/smail_gpl.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+short_name: SMAIL GPL
+name: SMAIL GENERAL PUBLIC LICENSE
+category: Copyleft
+owner: Landon Curt Noll & Ronald S. Karr
+homepage_url: https://gitweb.gentoo.org/repo/gentoo.git/plain/licenses/
```
Comment (pombredanne) [inline src/licensedcode/data/licenses/smail_gpl.yml]: it is ok here not to have a homepage so I would remove this rather than to make it up

**Comment by pombredanne** [comment]
Looking good. Merging! thank you :heart:

**Pull request merged**